### DISCUSSION TITLE
Welcome to pull-request Discussions!

### DISCUSSION THREAD

**Discussion by learningssik** [learningssik/pull-request#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wodnjse** [comment]
Hi! I read your post. Can I get help getting a Galaxy Brain badge?

**Reply by learningssik** [reply]
#20 #21 에 들어가셔서 코멘트 다시면 됩니다!
이 외에도 직접 오픈소스에 들어가셔서 채택을 받으시면 동 -> 은 -> 금으로 badge를 업그레이드할 수 있습니다.
